sky/clouds/service_catalog/fluidstack_catalog.py

```diff
@@ -0,0 +1,129 @@
+"""Fluidstack Cloud Catalog.
+
+This module loads the service catalog file and can be used to query
+instance types and pricing information for FluidStack.
+"""
+import typing
+from typing import Dict, List, Optional, Tuple
+
+from sky.clouds.service_catalog import common
+from sky.utils import ux_utils
+
+if typing.TYPE_CHECKING:
+    from sky.clouds import cloud
+
+_df = common.read_catalog('fluidstack/vms.csv')
+
+# Number of vCPUS for gpu_1x_a10
+_DEFAULT_NUM_VCPUS = 30
```

Comment:
How about changing this to 6 instead? Otherwise, `sky launch` will try to launch `RTXA4000:8` while we have cheaper machines with a single `RTXA4000` and 6 CPU cores.